deploy/src/main/deploy/example-credentials.json

```diff
@@ -49,7 +49,7 @@
       {
         "device-id": "4711",
         "type": "x509-cert",
-        "auth-id": "${cert.device-4711.subject}",
+        "auth-id": "CN=Device 4711,OU=Hono,O=Eclipse IoT,L=Ottawa,C=CA",
```

Comment:
IMHO we can the remove the example-credentials, -tenants and -device-identities altogether. They are not being deployed by the helm chart and we do not rely on nor use them in the getting started guide, do we?

Reply:
The example files are only referenced in the _admin_ guide of the file-based device registry.

Reply:
Meaning?

Reply:
I would wait with removing the example configurations until the file-based registry is (finally!) removed. Otherwise, we need to fix the links in the admin guide.

Reply:
Ok, so we'll keep the example files for reference for the time being ...